docs/csharp/language-reference/operators/snippets/patterns/DeclarationAndTypePatterns.cs

```diff
@@ -12,17 +12,15 @@ public static void Examples()
 
         private static void NotNullPattern()
         {
-            object input = default;
-            // <NotNull>
+            object input = new object();
+            // <NonNullCheck>
             if (input is not null)
             {
-                Console.WriteLine(input);
-            } else
-            {
-                throw new ArgumentNullException(paramName: nameof(input), message: "Input should not be null");
+                // ...
             }
-            // </NotNull>
+            // </NonNullCheck>
```

Comment:
For consistency, using the same example for `not null` as in other sections of the patterns reference

Reply:
I like that.